docs/contributors/contributors-guide.md

```diff
@@ -0,0 +1,163 @@
+2. **Connect individual repositories**:  Connecting individual repositories is a good idea if you who wants to keep track the contributions of a specific repository or group of repositories. For example, you might want to track of projects you have contributed to or would like to contribute to in the future.
+
+![sync insights gif](../../static/gif/insight-page-demo.gif)
+
+### Using Your Insights Page
+
+Once you have connected your repositories, you will be redirected to your Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
+
+There are three tabs that provide you with more information on these repositories:
+
+#### Reports
+
+The Reports tab allows paid users to filter for more information about their repositories over the last 30 days and to generate a Downloadable CSV. Filters include: Top 100 Repos, Minimum 5 Contributors, Recent, Most Active, and Spammed. This can be helpful if you're a beginner contributor who is looking for mentors or someone who is looking for potential maintainers for your project.
+
+#### Repositories
+
+If you're looking for an active project to contribute to, this tab can help you as it contains information such as Activity, PR Overview, PR Velocity, Spam, Contributors, and activity that has occurred over the last 30 days. This information is also a good way to connect with others on the project or where to find support.  To learn more about these features, see [Understanding Insights Data](https://docs.opensauced.pizza/maintainers/understanding-lists/#understanding-lists-contributor-data).
+
+![repositories tab](../../static/img/repos-insights.png)
+
+#### Contributors
+
+The Contributors tab allows you to view more detailed information about other contributors, including Activity, Repositories, the date of their last contribution, Time Zone, and the number of contributions. This can be a great way to find other people to collaborate with or recommend for jobs.
+
+Want to learn more about your peers?  You can select them and add them to a list.
+
+![contributors insight list](../../static/img/contributors-insights.png)
+
+There are two ways to add contributors to a list:
+
+1. Create a new list with selected contributors.
+2. Add selected contributors to an existing list.
+
+## Lists: Connecting with Contributors
+
+[Lists](../features/lists.md) feature helps you connect with other contributors or learn more about what others are up to in open source. With this feature, you can categorize, watch, and check out different groups of contributors within open source projects.
+
+### What You Can Do With Lists?
+
+There are many things you can do with this feature, but here's some ideas to get you started:
+
+- Learn about what other contributors are doing in open source.
+- Learn about new projects.
+- Make connections with other contributors who share your interests and come from other backgrounds.
```

Comment:
Do you mean here come from different or same backgrounds?

```suggestion
- Connect with other contributors who share your interests and come from other backgrounds.
```

Reply:
Hi @adiati98. I'm trying to say that this feature can help contributors connect with people who share similar interests mostly.